xmodule/conditional_block.py

```diff
@@ -213,7 +213,7 @@ def is_condition_satisfied(self):  # lint-amnesty, pylint: disable=missing-funct
             for block in self.get_required_blocks:
                 if not hasattr(block, attr_name):
                     # We don't throw an exception here because it is possible for
-                    # the descriptor of a required block to have a property but
+                    # the block of a required block to have a property but
                     # for the resulting module to be a (flavor of) ErrorBlock.
```

Comment:
```suggestion
                    # the required block to have a property but
                    # for the resulting block to be a (flavor of) ErrorBlock.
```

An alternative would be changing the `descriptor` to `runtime`, but we don't need to be that specific here.

Reply:
updated